### ISSUE TITLE
Turning off echo doesn't work

### ISSUE THREAD

**Issue by jhudsoncedaron** [PowerShell/Win32-OpenSSH#143]
C:\Openssh-win64> ssh locahost
redacted@localhost's password:
redacted@w10dev C:\Users\redacted>cd \Openssh-win64
C:\Openssh-win64> ssh locahost
redacted@localhost's password: displays password on screen

NO! NO! NO! NO!

You're going to need some minimal terminal emulation so echo on/off doesn't work.

Since Windows can't do proper terminal emulation, maybe it needs to implement the obscure ^[[12h and ^[[12l for echo off/on. Ref: http://www.real-world-systems.com/docs/ANSIcode.html

**Comment by quamrulmina** [comment]
This problem has now been fixed and will be propagated when the next release is out. Before echoing in a session, the sshd server checks if echo mode is on. Will work for ssh, sftp, scp, net.exe and all apps tweaking echo mode off in the console when they take a password.

**Comment by manojampalam** [comment]
This should not be the case with latest release. Reopen if you see otherwise

**Comment by jhudsoncedaron** [comment]
Had you checked the history you would see the change got reverted.
https://github.com/PowerShell/Win32-OpenSSH/commit/69546ae1ff256e9c7b68379cd63fc77932332304

In testing it, terminal handling is now hosed really badly when I used putty as the client and even when using the windows cmd as the terminal, the nested ssh now yields "Pseudo-terminal will not be allocated because stdin is not a terminal".

It won't let me reopen.
